pkg/cosign/verify_sct_test.go

```diff
@@ -40,10 +43,18 @@ func TestValidateAndUnpackCertWithSCT(t *testing.T) {
 
 	rootPool := x509.NewCertPool()
 	rootPool.AddCert(chain[1])
+
+	// Grab the CTLog public keys
+	pubKeys, err := ctl.GetCTLogPubs(context.Background())
```

Comment:
There's no keys at this point set in the environment, so I think you only need the call that's already done below. You can just not set them in CheckOpts yet

Reply:
Previous code would have fetched them from the TUF root deep in the bowels, so the error messages were differing (CTFE keys not found vs. what's expected here). Tried to keep the behaviour consistent by not changing that. But, I can change it if you feel strongly?

Reply:
I'm just wondering what this is testing for. This is fetching the default set of keys from the embedded TUF root, and then we override this set of keys by fetching them again, but from the disk. Given we just want to set up the correct set of keys at this point, I think we can just rely on the call below to initialize the keyset from the disk keys.

Reply:
nm, you're totes right. sorry, too many of these areound :)